src/wallet/wallet2.cpp

```diff
@@ -4867,27 +4876,16 @@ bool wallet2::sign_tx(unsigned_tx_set &exported_txs, const std::string &signed_f
   return true;
 }
 //----------------------------------------------------------------------------------------------------
-bool wallet2::load_tx(const std::string &signed_filename, std::vector<tools::wallet2::pending_tx> &ptx, std::function<bool(const signed_tx_set&)> accept_func)
```

Comment:
load_tx_from_file would be better but I did not want to break backward compatibility of wallet2.

Reply:
AIUI there's no such thing as backward compatibility for wallet2, unlike the wallet API. Renaming it to `load_tx_from_file` seems fine to me.